doc/man3/SSL_set_async_callback.pod

```diff
@@ -0,0 +1,101 @@
+=pod
+
+=head1 NAME
+
+SSL_CTX_set_async_callback,
+SSL_CTX_set_async_callback_arg,
+SSL_set_async_callback,
+SSL_set_async_callback_arg,
+SSL_get_async_status,
+SSL_clear_async_status,
+SSL_async_callback_fn
+- manage asynchronous operations
+
+=head1 SYNOPSIS
+
+=for comment multiple includes
+
+ #include <openssl/ssl.h>
+
+ typedef int (*SSL_async_callback_fn)(SSL *s, void *arg);
+ int SSL_CTX_set_async_callback(SSL_CTX *ctx, SSL_async_callback_fn callback);
+ int SSL_CTX_set_async_callback_arg(SSL_CTX *ctx, void *arg);
+ int SSL_set_async_callback(SSL *s, SSL_async_callback_fn callback);
+ int SSL_set_async_callback_arg(SSL *s, void *arg);
+ int SSL_get_async_status(SSL *s, int *status);
+ int SSL_clear_async_status(SSL *s);
+
+=head1 DESCRIPTION
+
+SSL_CTX_set_async_callback() sets asynchronous callback function, and all SSL
```

Comment:
"... sets *an* asynchronous callback function. All SSL sessions generated based on this SSL_CTX will get this callback. It will be automatically called when *an* engine completes a cryptography operation..."